src/test/regress/expected/local_shard_execution.out

```diff
@@ -398,7 +398,9 @@ INSERT INTO distributed_table VALUES (1, '22', 20);
 NOTICE:  executing the command locally: INSERT INTO local_shard_execution.distributed_table_1470001 (key, value, age) VALUES (1, '22'::text, 20)
 INSERT INTO second_distributed_table VALUES (1, '1');
 NOTICE:  executing the command locally: INSERT INTO local_shard_execution.second_distributed_table_1470005 (key, value) VALUES (1, '1'::text)
+SET citus.enable_ddl_propagation TO OFF;
```

Comment:
Can you please add comments (over github is fine for now) why we disable each view test? It is really unclear by a quick look

Reply:
We've discussed this online. We've disabled ddl propagation because the view is only created on the worker within tests.